src/app/util/util.cpp

```diff
@@ -289,6 +289,27 @@ void emberAfTick(void)
 #endif
 }
 
+// Cluster init functions that don't have a cluster implementation to define
+// them in.
+void MatterBooleanStatePluginServerInitCallback() {}
```

Comment:
complaining about 'util' naming of this file here ... these InitCallbacks seem to have little relationship with  other things inside this file but `util` name is  a grab bag for 'anything fits'.

Could we maybe create a separate `callback-stubs.cpp` file?

Reply:
We could, but then we would need to add it to the build system in a bunch of places, which I was sort of trying to avoid.

Also, we already have a generated `callback-stub.cpp` file, which would make adding another "stubs" file a little odd too .... but that's generated per-app, so adding these there is not great either.

Reply:
Oh, and I put them in this file because the callsite for them is in this file, in the function immediately above where I added them.

Reply:
ok. things to improve later.